src/covid_shared/cli_tools.py

```diff
@@ -29,6 +29,20 @@
 }
 
 
+def _remove_secrets(dictionary: Dict, secrets: List[str]) -> Dict:
```

Comment:
Your secrets type is `Optional[List[str]]` based on how it's called.

Reply:
Also, make a static method of RunMetadata so it doesn't show up as a free-floating method on `cli_tools` with tab completion.